docs/src/use/configure/configuration-files.md

```diff
@@ -342,6 +343,57 @@ Here, the `js.configs.recommended` predefined configuration is applied first and
 
 For more information on how to combine predefined configs with your preferences, please see [Combine Configs](combine-configs).
 
+### Configuration Naming Conventions
+
+The `name` property is optional, but it is recommended to provide a name for each configuration object, especially when you are creating shared configurations. The name is used in error messages, and the config inspector to help identify which configuration object is being used.
+
+The name should be descriptive of the configuration object's purpose and scoped with the configuration name or plugin name. Using `/` as the separator. ESLint does not enforce the names to be unique at runtime, but it is recommended that unique names be set to avoid confusion.
```

Comment:
```suggestion
The name should be descriptive of the configuration object's purpose and scoped with the configuration name or plugin name using `/` as a separator. ESLint does not enforce the names to be unique at runtime, but it is recommended that unique names be set to avoid confusion.
```